Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultTransformToolOptionsView.kt

```diff
@@ -0,0 +1,162 @@
+/*
+ * Paintroid: An image manipulation application for Android.
+ * Copyright (C) 2010-2021 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.paintroid.ui.tools
+
+import android.text.Editable
+import android.text.InputFilter
+import android.text.TextWatcher
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.SeekBar
+import android.widget.SeekBar.OnSeekBarChangeListener
+import androidx.appcompat.widget.AppCompatEditText
+import androidx.appcompat.widget.AppCompatSeekBar
+import androidx.appcompat.widget.AppCompatTextView
+import org.catrobat.paintroid.R
+import org.catrobat.paintroid.tools.options.TransformToolOptionsView
+import java.text.NumberFormat
+import java.text.ParseException
+import java.util.Locale
+
+private const val HUNDRED = 100
+
+class DefaultTransformToolOptionsView(rootView: ViewGroup) : TransformToolOptionsView {
+    private val heightTextWatcher: TransformToolSizeTextWatcher
+    private val widthTextWatcher: TransformToolSizeTextWatcher
+    private val widthEditText: AppCompatEditText
+    private val heightEditText: AppCompatEditText
+    private val resizeSeekBar: AppCompatSeekBar
+    private val percentageText: AppCompatTextView
+    private var callback: TransformToolOptionsView.Callback? = null
+
+    companion object {
+        private val TAG = DefaultTransformToolOptionsView::class.java.simpleName
+    }
```

Comment:
Only thing i could think of is making a compile time constant instead of the companion object. as the TAG is always "DefaultTransformToolOptionsView". What do you think about that? Rest LGTM